TodoWithControllersAuthJWT/Program.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Todos
+{
+    class Program
+    {
+        public static string ISSUER = "iamissuer";
+        public static string AUDIENCE = ISSUER;
+        public static byte[] KEY = Enumerable.Range(0, 100).Select(Convert.ToByte).ToArray();
+
+        public static Dictionary<string, string>  validUsers = new Dictionary<string, string>
```

Comment:
This should be a service injected into the controller with an IsValidUser(string username, string password)